framework/python/setup.cfg

```diff
@@ -0,0 +1,19 @@
+[metadata]
+name = benchpress
+version = 0.0.1
+author = Uffaz Nathaniel
+url = https://github.com/Azure/benchpress
+description = Testing framework for Azure deployment features by using Bicep.
+keywords = azure, bicep, arm, iac, testing
+
+[options]
+package_dir=
+    =src
```

Comment:
It this line correct?  `=` without a value on the left hand side?

Reply:
Yes this is correct. The `package_dir` accepts a dictionary. In older formats of setup.cfg and Python code, you may see code like this:
```
from setuptools import find_packages
setup(
...
    package_dir={'': 'src'}
    packages=find_packages(where='src')
...
)
```
Notice the `package_dir` takes a dictionary where the key is empty. The absence of left-side is the empty string and right-hand side is value *src*.